gateway/src/variant/dicl.rs

```diff
@@ -256,6 +260,18 @@ impl Variant for DiclConfig {
     fn get_all_template_paths(&self) -> Vec<&PathBuf> {
         vec![]
     }
+
+    async fn start_batch_inference<'a>(
+        &'a self,
+        _input: &[Input],
+        _models: &'a InferenceModels<'a>,
+        _function: &'a FunctionConfig,
+        _inference_configs: &'a [InferenceConfig<'a, 'a>],
+        _clients: &'a InferenceClients<'a>,
+        _inference_params: Vec<InferenceParams>,
+    ) -> Result<BatchModelInferenceWithMetadata<'a>, Error> {
+        Err(ErrorDetails::UnsupportedVariantForBatchInference { variant_name: None }.into())
+    }
```

Comment:
Create a TODO/issue for this. This is doable in batch format.

Reply:
How?

Reply:
presumably it's not a good solution to embed each input in the batch synchronously?

are we gonna do multistep polling? I think that's not something we should kill ourselves over

Reply:
It's a reasonable starter solution. Embedding is often much much cheaper. But just TODO it.